src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -198,10 +198,10 @@
 		private String mimeType;
 		private String videoCodec;
 		private String supportLine;
-		/** List of embedded subs supported by renderer defined in the Supported line in the renderer.conf */
-		private String embeddedSubs;
-		/** List of external subs supported by renderer defined in the Supported line in the renderer.conf */
-		private String externalSubs;
+		// List of embedded subs supported by renderer defined in the Supported line in the renderer.conf
+		private String supportedEmbeddedSubtitlesFormats;
+		// List of external subs supported by renderer defined in the Supported line in the renderer.conf
```

Comment:
This change means it's no longer a JavaDoc, and thus won't show up when you hover over the variable in IDE's etc.

Reply:
That's true, but I almost removed those comments since the new variable name already describes it IMO